aten/src/ATen/native/mkldnn/MKLDNNConversions.cpp

```diff
@@ -33,6 +34,41 @@ Tensor dense_to_mkldnn(const Tensor& cpu_tensor) {
   return mkldnn_tensor;
 }
 
+// Mkldnn tensor has special non-public format for conv2d weights
+// (dense_to_mkldnn only converts dense tensor to mkldnn tensor with
+// public format). Ideep conv kernel will do implicit reorder if the
+// weight is not already in this optimized format. By the time I'm
```

Comment:
Thanks for writing this comment!

By the time -> At the time of writing this note,